kernel-laws/src/main/scala/cats/kernel/laws/IsSerializable.scala

```diff
@@ -19,7 +19,9 @@ private[laws] object IsSerializable {
       import java.io._
       val baos = new ByteArrayOutputStream()
       val oos = new ObjectOutputStream(baos)
+      // scalastyle:off null
       var ois: ObjectInputStream = null
+      // scalastyle:off on
```

Comment:
Btw, it seems that you can also disable scalastyle on single lines with
```scala
var ois: ObjectInputStream = null // scalastyle:ignore null
```
see http://www.scalastyle.org/configuration.html.